export/export.go

```diff
@@ -339,7 +340,15 @@ func (s *Service) listPcs(ctx context.Context, r *ResourcesForExport, eg *util.E
 			klog.Errorf("failed to call list priorityClasses: %v", err)
 			pcs = &schedulingv1.PriorityClassList{Items: []schedulingv1.PriorityClass{}}
 		}
-		r.PriorityClasses = pcs.Items
+		result := []schedulingv1.PriorityClass{}
+		// The name of a PriorityClass object cannot be prefixed with `system-`.
+		// It is reserved by the system and we cannot recreate it. No need to export.
+		for _, i := range pcs.Items {
+			if !strings.HasPrefix(i.GetObjectMeta().GetName(), "system-") {
```

Comment:
Is it better to filter out the exact match of `system-cluster-critical` and `system-node-critical`? In case users create `system-xxx` PriorityClass by themselves.

Reply:
Users cannot create `system-xxx` PriorityClasses. So, it looks good.

> The name of a PriorityClass object must be a valid [DNS subdomain name](https://kubernetes.io/docs/concepts/overview/working-with-objects/names#dns-subdomain-names), and it cannot be prefixed with system-.
https://kubernetes.io/docs/concepts/scheduling-eviction/pod-priority-preemption/#priorityclass